code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -371,3 +371,52 @@
+			 Your attempt to fly away was halted when Nar-Sie somehow was able to start attempting to drag the transport into her grasp. In one final desperate move, the pilots turnd the ship around \
+			 and ramed it directly into Nar-Sie's head, her screams somehow echoed throughout the transport as she released her grasp on the transport, unfortunately taking half the engines with her. \
+			 Your final thoughts were panic as you quickly dove into your cryosleep pod, hoping you'll survive the crash landing.</b>"
+		if(2)
+			flavour_text += "Panic and paranoia ran high as a group of changelings somehow infiltrated the Inheritor. Utterly unprepared to deal with this threat, the crew tore itself apart. \
+			In a final desperate move to end the changeling threat, the ship's Commander activated the Inheritor's self destruct sequence and remotely shut down all escape pods. You and your \
+			squadmates, who were already on the brink of a psychological breakdown, made a run for the last remaining heavy troop transport. You arrive to discover the changelings had a similar plan \
+			, with minutes until the ship exploded, the changelings abandoned any pretense of subtly and revealed their true forms. After a intense gun fight you and your squad were able to board \
+			the transport. However as you take off, one of the surviving Changelings was able to plant an explosive charge on your engines. The explosion disabled your ship, and you got caught in a \
+			nearby planets gravity well. The survivors were forced into cryo sleep, as you're last thoughts were hoping to god none of your squad mates were secretly changelings.</b>"
+		if(3)
+			flavour_text += "The deeper reaches of space are home to hoards of pirates and marauders, usually they are no threat to a Earth capital ship. However, the Inheritor had conducted a \
+			large series of anti-piracy raids in your assigned sector, so crippling that every pirate in the local sector network formed a massive fleet in a attempt to destroy the Inheritor. The \
+			Inheritor won against all odds, but it was a pyrrhic victory as the Inheritor was byond repair. The acting Commander ordered all hands to abandon ship before it was scuttled. You and \
+			your remaining squad members were assigned to troop transport 5. Unbeknown to you or your pilots, your fuel tanks had suffered a massive leak during the fighting, leaving you with minimal \
+			fuel, thus your pilots had no choice but to land on a nearby planet. Strapping into your cryochamber, you can only hope that the pilots could land this thing safely.</b>"
+		if(4)
+			flavour_text += "The Inheritor was assigned to a high level covert recon mission of Space Station 13, a Nanotrasen research and development station. Stipulation of the Nanotrasen - United \
+			Earth Government plasma mining contract was that Nanotrasen were to be limited to mining only single gas giant and its orbiting moons. Military Intelligence received information that \
```

Comment:
Kor has already stated he is fine with the lore as is. My lore entries are on the same level as lore entries for all other lavaland roles.